.github/workflows/tests_windows.yml

```diff
@@ -163,9 +93,111 @@ jobs:
           cmake --build ./Build --config Debug --verbose
           cmake --install ./Build --config Debug --verbose
 
+  build_dependencies_vcpkg:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest]
+    timeout-minutes: 30
+    name: vcpkg dependencies
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Cache installation directories
+        id: vcpkg-cache
+        uses: actions/cache@v4
+        with:
+          path: D:\a\install_dir\vcpkg
```

Comment:
nice! I guessed there is more work required for caching vcpkg when suggested to use that action :smile: the cached folder is quite large (830 MB)!